src/plugins/notebook/utils/notebook-snapshot-menu.js

```diff
@@ -0,0 +1,31 @@
+import { getDefaultNotebook, getDefaultNotebookObject, getNotebookSectionAndPage } from './notebook-storage';
+
+export async function getMenuItems(openmct, menuItemOptions) {
+    const notebookTypes = [];
+
+    const defaultNotebook = getDefaultNotebook();
+    const defaultNotebookObject = defaultNotebook && await getDefaultNotebookObject(openmct);
```

Comment:
Hrm, I'm not sure if we need a `getDefaultNotebookObject` function. We already have the default notebook metadata from `getDefaultNotebook` (which is perhaps what **that** function should be called). All it's doing is just wrapping a call to `openmct.objects.get()`

Reply:
removed that function